packages/source_highlight.rb

```diff
@@ -0,0 +1,47 @@
+# Adapted from Arch Linux source-highlight PKGBUILD at:
+# https://github.com/archlinux/svntogit-packages/raw/packages/source-highlight/trunk/PKGBUILD
+
+require 'package'
+
+class Source_highlight < Package
+  description 'Convert source code to syntax highlighted document'
+  homepage 'https://www.gnu.org/software/src-highlite/'
+  version '3.1.9-9049'
+  license 'GPL'
```

Comment:
Shouldn't we also supply the version?

Reply:
That's a mix of version from the source plus hash...